endpoints/openrtb2/auction_test.go

```diff
@@ -281,15 +187,36 @@ func assertBidResponseEqual(t *testing.T, testFile string, expectedBidResponse o
 		expectedSeatBidsMap[seatBid.Seat] = seatBid
 	}
 
-	for k, expectedSeatBid := range expectedSeatBidsMap {
+	for bidderName, expectedSeatBid := range expectedSeatBidsMap {
```

Comment:
For my own understanding, did you change this variable name here from `k` to `bidderName` just for clarity? Or was something structurally changed that required a name change.

Reply:
Yes. I was trying to make it more readable. `k` seemed to not be as approachable as `bidderName` to me.

Reply:
I think the name change to `bidderName` is great!